packages/admin-panel/src/table/DataFetchingTable/DataFetchingTable.jsx

```diff
@@ -180,7 +180,7 @@ const DataFetchingTableComponent = ({
     }
     gotoPage(0);
     setSortBy(defaultSorting ?? []); // reset sorting when table is re-initialised
-  }, [endpoint, baseFilter]);
+  }, [endpoint, JSON.stringify(baseFilter)]);
```

Comment:
`baseFilter` will be an object, and often if you pass in a nested object, checking the equality causes infinite re-renders and bugs. So stringifying it will stop that happening